style/infra_template.rubocop.yml

```diff
@@ -0,0 +1,87 @@
+## Infrastructure .rubocop.yml template
+
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  TargetRubyVersion: 3.1
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
+    - './Rakefile'
+    - './config.ru'
+    - '**/*.rb'
+    - '**/*.rake'
+  Exclude:
+    # autogenerated files (that we don't change) should go here
+    - 'bin/**/*'
+    - 'config/environments/*.rb'
+    - 'db/**/*'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+    - 'vendor/**/*'
+
+Bundler/GemComment:
+  OnlyFor:
+    - restrictive_version_specifiers
+    - branch
+
+# Configuration parameters: AllowURI, URISchemes.
+Layout/LineLength:
+  Max: 120
+
+Metrics/BlockLength:
+  IgnoredMethods:
+    - SimpleForm.setup
+    - FactoryBot.define
+    - factory
+    - describe
+    - context
+    - config/routes.rb
```

Comment:
Should this be in a separate section for ignored files as this is not a method?

Reply:
thanks - it should be elsewhere.  I will fix.